packages/components/src/radio-group/stories/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Radio from '../../radio';
+import RadioGroup from '../';
+
+export default { title: 'Components/RadioGroup', component: RadioGroup };
+
+export const _default = () => {
+	/* eslint-disable no-restricted-syntax */
+	return (
+		<RadioGroup
+			// id is required for server side rendering
+			id="default-radiogroup"
```

Comment:
We might start inlining the comment in id 😂

id=“required-for-server-side-rendering-only”?